### DISCUSSION TITLE
2.9.0安装报错，哪位大佬帮忙解决一下，看看报错。

### DISCUSSION THREAD

**Discussion by melinetssamant** [apache/kvrocks#2453] [Q&A]
报错信息如下：
[root@middleware kvrocks]# ./x.py build
which: no cmake in (/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/root/bin)
Traceback (most recent call last):
  File "/opt/kvrocks/./x.py", line 448, in <module>
    args.func(**arg_dict)
  File "/opt/kvrocks/./x.py", line 120, in build
    cmake = find_command(cmake_path, msg="CMake is required")
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/kvrocks/./x.py", line 80, in find_command
    return run_pipe("which", command, msg=msg).read().strip()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/kvrocks/./x.py", line 75, in run_pipe
    p = run(*args, msg=msg, verbose=verbose, stdout=PIPE, universal_newlines=True, **kwargs)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/kvrocks/./x.py", line 69, in run
    raise RuntimeError(err)
RuntimeError:
failed to run: ('which', 'cmake')
exit with code: 1
error message: CMake is required

**Comment by tisonkun** [comment]
> which: no cmake in (/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/root/bin)

字面意思，你的机器上没有安装 `cmake`

Check https://cmake.org/download/ or https://askubuntu.com/questions/355565/how-do-i-install-the-latest-version-of-cmake-from-the-command-line
